.github/actions/gradle-setup/action.yml

```diff
@@ -25,6 +25,13 @@ runs:
         path: IdentityHub
         ref: 76234800e5ebd7ccfb97aeab305a34961f50cdec
 
+    - name: Checkout EDC Data Dashboard
+      uses: actions/checkout@v2
+      with:
+        repository: eclipse-dataspaceconnector/DataDashboard
+        path: DataDashboard
+        ref: ab1b6fd0870fa5ec8c170915a2b44fae63d576bd
+
```

Comment:
with this we can also update Build-Datadasboard step checkout https://github.com/agera-edc/MinimumViableDataspace/blob/65c4c6b49bc6c8fca8ed875cb7a2e3b83a5d6b9e/.github/workflows/deploy.yaml#L107.

what do you think ?

Reply:
Good catch! Will remove the redundant checkout.